### PR TITLE
chore: nit on messages

### PR DESCRIPTION
**Pull request by eordano**

### CODE REVIEW

File: init.sh [hunk 1]
```diff
@@ -227,27 +228,31 @@ if [ ${CATALYST_URL} != "http://localhost" ]; then
     # This is the URL without the 'http/s'
     # Needed to place the server on nginx conf file
     if [ -d "$data_path/conf/live/$nginx_url" ]; then
-    echo -n "## Existing data found for $nginx_url. "
+    echo -n "Existing data found for $nginx_url. "
+    printMessage ok
```
Comment (eordano) [inline init.sh]:
```suggestion
    printMessage ok
    fi
```

File: init.sh [hunk 2]
```diff
@@ -267,5 +272,4 @@ if test $? -ne 0; then
   printMessage failed
   exit 1
 fi
-echo -n "## containers started Ok... "
-printMessage ok
\ No newline at end of file
+echo "## Catalyst server is up and running at $katalyst_host"
```
Comment (eordano) [inline init.sh]:
```suggestion
echo "## Catalyst server is up and running at \$katalyst_host"
```

File: init.sh [hunk 3]
```diff
@@ -227,27 +228,31 @@ if [ ${CATALYST_URL} != "http://localhost" ]; then
     # This is the URL without the 'http/s'
     # Needed to place the server on nginx conf file
     if [ -d "$data_path/conf/live/$nginx_url" ]; then
-    echo -n "## Existing data found for $nginx_url. "
-    if test ${REGENERATE} -eq 1; then
-        leCertEmit $nginx_url
-    else
-        echo -n "## Keeping the current certs"
-    fi
+        echo -"Existing data found for $nginx_url."
```
Comment (eordano) [inline init.sh]:
```suggestion
        echo "Existing data found for \$nginx_url."
```

File: init.sh [hunk 4]
```diff
@@ -221,33 +220,35 @@ fi
 # else, create new certs
 export nginx_url=`echo "${CATALYST_URL##*/}"`
 if [ ${CATALYST_URL} != "http://localhost" ]; then
-    echo -n "## Replacing HTTPS \$katalyst_host on nginx server file ${nginx_url}... "
+    echo -n "## Replacing HTTPS $katalyst_host on nginx server file ${nginx_url}... "
```
Comment (eordano) [inline init.sh]:
```suggestion
    echo -n "## Replacing HTTPS \$katalyst_host on nginx server file ${nginx_url}... "
```

File: init.sh [hunk 5]
```diff
@@ -267,5 +270,4 @@ if test $? -ne 0; then
   printMessage failed
   exit 1
 fi
-echo -n "## containers started Ok... "
-printMessage ok
\ No newline at end of file
+echo "## Catalyst server is up and running at \$katalyst_host"
```
Comment (eordano) [inline init.sh]:
```suggestion
echo "## Catalyst server is up and running at $katalyst_host"
```

**Comment by adrabenche** [comment]
Testing changes...

**Pull request merged**